openvmm/openvmm_entry/src/lib.rs

```diff
@@ -1427,8 +1433,13 @@ fn disk_open(disk_cli: &DiskCliKind, read_only: bool) -> anyhow::Result<Resource
             },
         }),
         DiskCliKind::MemoryDiff(inner) => {
-            Resource::new(disk_backend_resources::RamDiffDiskHandle {
-                lower: disk_open(inner, true)?,
+            Resource::new(disk_backend_resources::LayeredDiskHandle {
+                layers: vec![
+                    RamDiskLayerHandle { len: None }.into_resource().into(),
```

Comment:
to my eyes, I would've expected to see some kind of indication that the `RamDiskLayer` was implementing diff-disk semantics here (e.g: some sort of bool to configure the semantics). instead - it kinda just looks like you have a regular ram disk on-top of a underlying disk (i.e: a useless configuration)

Reply:
A RAM disk layer starts out with all sectors "not present", not zero. When a read falls off the end of a layered disk (because the sector was not present in any of the layers), we zero out the sector.

Reply:
I see. I just need to get used to the different semantics that Layers have, vs. typical Disks